app/__init__.py

```diff
@@ -241,7 +241,7 @@ def update_sent_state(sender=None, headers=None, **kwargs):
 scheduler.add_job(send_event_fee_notification, 'cron', day=1)
 scheduler.add_job(send_event_fee_notification_followup, 'cron', day=15)
 scheduler.add_job(change_session_state_on_event_completion, 'cron', hour=5, minute=30)
-scheduler.add_job(expire_pending_tickets_after_three_days, 'cron', hour=5)
+scheduler.add_job(expire_pending_tickets_after_30_min, 'cron', hour=1)
```

Comment:
I think we should increase the frequency . From 1 hour to 15-30 minutes.

Reply:
@uds5501 Yeah, I was a little skeptical about that matter. I think 30 would do.

Reply:
@mrsaicharan1 Yes, exactly that. I think 30 would be fine